ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__v4.13.yaml

```diff
@@ -0,0 +1,53 @@
+base_images:
```

Comment:
@anandrkskd Can we add the 4.13 periodic tests into a separate PR instead? To me, it seems quite unrelated to this PR (about not running odo integration tests on Prow).
It will also help us track the changes later.

Reply:
sure, I have removed the 4.13 tests from this PR.